<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konwersja do odcieni szarości</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="image"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="bOpenFile"/>
    <addaction name="bSaveFile"/>
    <addaction name="separator"/>
    <addaction name="bOpenConf"/>
    <addaction name="bSaveConf"/>
    <addaction name="separator"/>
    <addaction name="bClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="bHelp"/>
    <addaction name="bAbout"/>
    <addaction name="bAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="chanelsWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>165</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Mikser Kanałów</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QComboBox" name="chanelMode">
       <item>
        <property name="text">
         <string>Gimp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Naturalna</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="cwLayout" columnstretch="0,1,0" columnminimumwidth="0,100,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Czerwony</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Zielony</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Niebieski</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="blueSBox">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QScrollBar" name="blueSBar">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QScrollBar" name="greenSBar">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="greenSBox">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QScrollBar" name="redSBar">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="redSBox">
         <property name="minimum">
          <number>-200</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="duotoneWidget">
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>103</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Bichromia</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QComboBox" name="duotoneMode">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normalna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Odwrotna</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0" columnminimumwidth="0,100,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="1">
        <widget class="QScrollBar" name="hueSBarDt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="duotoneLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Odcień</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="hueSBoxDt">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="rangeWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Selekcja barwy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QComboBox" name="rangeMode">
       <item>
        <property name="text">
         <string>---</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brak redukcji odcieni</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Redukcja odcieni</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0" columnminimumwidth="0,100,0">
       <item row="0" column="1">
        <widget class="QScrollBar" name="hueSBarR">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="hueSBoxR">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>359</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="rangeLabel1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Odcień</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rangeLabel2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Tolerancja</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="tolSBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QScrollBar" name="tolSBar">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Mieszanie</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QGridLayout" name="gridLayout_5" columnstretch="0,1,0" rowminimumheight="0" columnminimumwidth="0,100,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item row="0" column="1">
      <widget class="QScrollBar" name="mergeSBar">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mieszanie</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="mergeSBox">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="bOpenFile">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="bUndo">
   <property name="text">
    <string>Cofnij</string>
   </property>
  </action>
  <action name="bRedo">
   <property name="text">
    <string>Ponów</string>
   </property>
  </action>
  <action name="bSaveFile">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="bOpenConf">
   <property name="text">
    <string>Wczytaj konfigurację</string>
   </property>
  </action>
  <action name="bSaveConf">
   <property name="text">
    <string>Zapisz konfigurację</string>
   </property>
  </action>
  <action name="bClose">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="bHelp">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
  <action name="bAbout">
   <property name="text">
    <string>O Programie</string>
   </property>
  </action>
  <action name="bAboutQt">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>bOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSaveFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bOpenConf</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bSaveConf</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveConf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>redSBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>greenSBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBarDt</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSBoxDt</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBoxDt</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSBarDt</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBoxDt</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blueSBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blueSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBarR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSBoxR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBoxR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueSBarR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSBarR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tolSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tolSBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tolSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tolSBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tolSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeSBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mergeSBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mergeSBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mergeSBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>491</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chanelMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>editImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rangeMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duotoneMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>openConf()</slot>
  <slot>saveFile()</slot>
  <slot>saveConf()</slot>
  <slot>editImage()</slot>
  <slot>changeMode()</slot>
  <slot>about()</slot>
  <slot>aboutQt()</slot>
  <slot>help()</slot>
 </slots>
</ui>
